### PR TITLE
Switching from Cobertura to JaCoCo

### PR DESCRIPTION
**Pull request by EricBeach**
tl:dr; - Cobertura does not play nice with Java 8, so switched to JaCoCo

The first sign of trouble with Cobertura was "WARNING No files found in report." in logs in Codecov.io (e.g., https://codecov.io/gh/GoogleCloudPlatform/pontem/commit/5c74ef21e30d0f8f3bef44521c34513c7af02047 ).

When running "mvn cobertura:cobertura" locally, I got a report but the report was blank. Logs said "Cobertura: Loaded information on 0 classes." Based upon a lot of looking and https://stackoverflow.com/questions/25293433/coberturainstrumenter-unable-to-instrument-file/25382674 I realized that Cobertura does not play nice with Java 8, so I switched to JaCoCo

== Codecov.io logs ==
May 15 13:50:20 [INFO] INFO GitHub HTTP 200 {"bot": "n/a", "commit": "5c74ef2", "endpoint": "/search/issues", "event": "api", "method": "GET", "public": "t", "rlr": "1526406680", "rlx": "29", "rly": "30", "service": "github", "slug": "GoogleCloudPlatform/pontem", "task": "upload"}
May 15 13:50:20 [INFO] INFO Retrieved report for processing. {"bot": "n/a", "commit": "5c74ef2", "public": "t", "service": "github", "slug": "GoogleCloudPlatform/pontem", "task": "upload", "url": "v4/raw/2018-05-15/4BF1A79D88D48CE6E7584A8A3B2D4547/5c74ef21e30d0f8f3bef44521c34513c7af02047/c78759c7-1be3-441a-9d3e-7787da8b762a.txt"}
May 15 13:50:20 [INFO] INFO Yaml discovered {"bot": "n/a", "commit": "5c74ef2", "path": ".codecov.yml", "public": "t", "service": "github", "slug": "GoogleCloudPlatform/pontem", "task": "upload"}
May 15 13:50:20 [INFO] INFO Yaml exists at current location. {"bot": "n/a", "commit": "5c74ef2", "public": "t", "service": "github", "slug": "GoogleCloudPlatform/pontem", "task": "upload"}
May 15 13:50:20 [WARNING] WARNING No files found in report. {"bot": "n/a", "commit": "5c74ef2", "public": "t", "service": "github", "slug": "GoogleCloudPlatform/pontem", "task": "upload"}

### CODE REVIEW

**Comment by EricBeach** [comment]
Fixes #44
